src/base/bittorrent/session.cpp

```diff
@@ -2368,6 +2368,36 @@ void Session::setIPFilterFile(QString path)
     }
 }
 
+void Session::setBannedIPs(const QStringList &newList)
+{
+    if (newList == m_bannedIPs)
```

Comment:
FYI, Qt has [qSort(Container & container)](https://doc.qt.io/qt-4.8/qtalgorithms.html#qSort-3) or if you wish use one of the std sorts...